src/webots/nodes/WbCompass.hpp

```diff
@@ -26,7 +26,7 @@ class WbCompass : public WbSolidDevice {
 public:
   // constructors and destructor
   explicit WbCompass(WbTokenizer *tokenizer = NULL);
-  WbCompass(const WbCompass &other);
+  explicit WbCompass(const WbCompass &other);
```

Comment:
Why here you define this constructor as `explicit` and not in the other class (e.g. `WbBrake.hpp` just before)?

Reply:
Because otherwise cppcheck will complain about it. Strangely it doesn't complain about it in other classes...

Reply:
Ok, thank you for the explanation, this sounds like a cppcheck bug.